dbms/src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -1273,6 +1273,40 @@ MergeTreeData::AlterDataPartTransactionPtr MergeTreeData::alterDataPart(
     return transaction;
 }
 
+void MergeTreeData::freezeAll(const String & with_name, const Context & context)
```

Comment:
Partial copy-paste of `MergeTreeData::freezePartition`. Can we do better?
Is it the same as if empty prefix is used inside `freezePartition`?

Reply:
Moved common part to a separate private helper method.